src/plasoscaffolder/frontend/controller/sqlite_controller.py

```diff
@@ -51,18 +60,21 @@ def SourcePath(self, ctx: click.core.Context, param: click.core.Option,
     """
     while not self._plugin_helper.FolderExists(value):
       value = self._output_handler.PromptError(
-          'Folder does not exists. Enter correct one: ')
+          'Folder does not exists. Enter correct one')
     self._path = value
```

Comment:
Why, I don't see it?
Value is set the first time and given in the callback.  
And while the condition is false, value be set to the new input the user gives until the condition is true.
I don't see how the first iteration is different.